test/integration_test/applicationbackup_test.go

```diff
@@ -968,7 +948,7 @@ func applicationBackupSyncControllerTest(t *testing.T) {
 	destroyAndWait(t, []*scheduler.Context{appCtx})
 }
 
-func waitForAppBackupCompletion(name, namespace string) error {
+func waitForAppBackupCompletion(name, namespace string, timeout time.Duration) error {
```

Comment:
timeout should be used in DoRetryWithTimeout(0 ?

Reply:
Fixed